addons/im_livechat/controllers/cors/main.py

```diff
@@ -30,7 +30,7 @@ def cors_email_livechat_transcript(self, guest_token, channel_id, email):
     def cors_get_session(
         self, channel_id, anonymous_name, previous_operator_id=None, chatbot_script_id=None, persisted=True, **kwargs
     ):
-        force_guest_env(kwargs.get("guest_token", ""), raise_if_not_found=False)
+        force_guest_env(kwargs.pop("guest_token", ""), raise_if_not_found=False)
```

Comment:
Note: not a consequence of this PR, but we should not pass this kwarg to the original route as it does not expect `guest_token` to be passed. As a result, a warning is logged by the http stack.